pkg/tsdb/tempo/trace_transform_test.go

```diff
@@ -0,0 +1,111 @@
+		// json. At the moment just saving some real tempo proto response into file and loading was the easiest and
+		// as my patience was diminished trying to figure this out, I say it's good enough.
+		proto, err := ioutil.ReadFile("testData/tempo_proto_response")
+		require.NoError(t, err)
+
+		otTrace := ot_pdata.NewTraces()
+		err = otTrace.FromOtlpProtoBytes(proto)
+		require.NoError(t, err)
+
+		frame, err := TraceToFrame(otTrace)
+		require.NoError(t, err)
+
+		require.Equal(t, 30, frame.Rows())
+		require.ElementsMatch(t, fields, fieldNames(frame))
+
+		bFrame := &BetterFrame{frame}
+		root := rootSpan(bFrame)
+		require.NotNil(t, root)
+
+		require.Equal(t, "HTTP GET - loki_api_v1_query_range", root["operationName"])
+		require.Equal(t, "loki-all", root["serviceName"])
+		require.Equal(t, "[{\"value\":\"loki-all\",\"key\":\"service.name\"},{\"value\":\"Jaeger-Go-2.25.0\",\"key\":\"opencensus.exporterversion\"},{\"value\":\"4d019a031941\",\"key\":\"host.hostname\"},{\"value\":\"172.18.0.6\",\"key\":\"ip\"},{\"value\":\"4b19ace06df8e4de\",\"key\":\"client-uuid\"}]", root["serviceTags"])
+		require.Equal(t, 1616072924070.497, root["startTime"])
+		require.Equal(t, 8.421, root["duration"])
+		require.Equal(t, "", root["logs"])
+		require.Equal(t, "[{\"value\":\"const\",\"key\":\"sampler.type\"},{\"value\":true,\"key\":\"sampler.param\"},{\"value\":200,\"key\":\"http.status_code\"},{\"value\":\"GET\",\"key\":\"http.method\"},{\"value\":\"/loki/api/v1/query_range?direction=BACKWARD\\u0026limit=1000\\u0026query=%7Bcompose_project%3D%22devenv%22%7D%20%7C%3D%22traceID%22\\u0026start=1616070921000000000\\u0026end=1616072722000000000\\u0026step=2\",\"key\":\"http.url\"},{\"value\":\"net/http\",\"key\":\"component\"},{\"value\":\"server\",\"key\":\"span.kind\"},{\"value\":0,\"key\":\"status.code\"}]", root["tags"])
+
+		span := bFrame.FindRowWithValue("spanID", "8185345640360084998")
+
+		require.Equal(t, "GetParallelChunks", span["operationName"])
+		require.Equal(t, "loki-all", span["serviceName"])
+		require.Equal(t, "[{\"value\":\"loki-all\",\"key\":\"service.name\"},{\"value\":\"Jaeger-Go-2.25.0\",\"key\":\"opencensus.exporterversion\"},{\"value\":\"4d019a031941\",\"key\":\"host.hostname\"},{\"value\":\"172.18.0.6\",\"key\":\"ip\"},{\"value\":\"4b19ace06df8e4de\",\"key\":\"client-uuid\"}]", span["serviceTags"])
+		require.Equal(t, 1616072924072.852, span["startTime"])
+		require.Equal(t, 0.094, span["duration"])
+		require.Equal(t, "[{\"timestamp\":1616072924072.856,\"fields\":[{\"value\":1,\"key\":\"chunks requested\"}]},{\"timestamp\":1616072924072.9448,\"fields\":[{\"value\":1,\"key\":\"chunks fetched\"}]}]", span["logs"])
+		require.Equal(t, "[{\"value\":0,\"key\":\"status.code\"}]", span["tags"])
+	})
+}
+
+type Row map[string]interface{}
+type BetterFrame struct {
+	frame *data.Frame
+}
+
+func (f *BetterFrame) GetRow(index int) Row {
```

Comment:
@kylebrandt @ryantxu can I add something like this to the Frame implementation in the SDK? Personally I rarely need specific ordering of columns and remembering that or searching for the column by name very time is tedious.